ntoskrnl/mm/section.c

```diff
@@ -5197,10 +5302,12 @@ MmCheckDirtySegment(
     return FALSE;
 }
 
+/* This function is not used. It is left for reference only */
```

Comment:
Is this reference actually needed? Do we badly need to keep the dead code in this case? Maybe it makes sense to remove it? :thinking:

Reply:
OK, then it probably makes sense to keep it as-is.

Reply:
> it's the per-process address space thing that was an option in these functions

Yeah, that's the point.